test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Rendering/HtmlHelperDisplayExtensionsTest.cs

```diff
@@ -94,6 +96,94 @@ public void Display_UsesTemplateName()
             viewEngine.Verify();
         }
 
+        public static TheoryData<FormatModel, string> DisplayFormatModels
```

Comment:
`EnumFormatModels` and reuse this to create a similar test in `HtmlHelperEditorExtensionsTest`.